substrate/frame/staking-async/src/tests/slashing.rs

```diff
@@ -611,18 +616,7 @@ fn garbage_collection_after_slashing() {
 			// Non staked balance is not touched.
 			assert_eq!(asset::total_balance::<T>(&11), ExistentialDeposit::get());
 
-			let slashing_spans = SlashingSpans::<T>::get(&11).unwrap();
```

Comment:
Similarly, is this test conveying anything of value now? it def should be renamed :D

Reply:
we are now checking that a fully-slashed staker can be repead?

Reply:
repurposed it to `fully_slashed_account_can_be_reaped`